presto-hive/src/test/java/com/facebook/presto/hive/parquet/ParquetTester.java

```diff
@@ -314,6 +397,11 @@ private static void assertFileContents(
     }
 
     private static void assertPageSource(List<Type> types, Iterator<?>[] valuesByField, ConnectorPageSource pageSource)
+    {
+        assertPageSource(types, valuesByField, pageSource, Optional.empty());
+    }
+
+    private static void assertPageSource(List<Type> types, Iterator<?>[] valuesByField, ConnectorPageSource pageSource, Optional<Long> maxReadBlockSize)
     {
         Page page;
         while ((page = pageSource.getNextPage()) != null) {
```

Comment:
@achopragh : Looks like this change is missing? https://github.com/prestosql/presto/pull/58/files#diff-af0230f49180fa0b1814f3f564aa6eb7R411

Reply:
Fixed